src/View/Helper/UrlHelper.php

```diff
@@ -146,7 +146,7 @@ public function script($path, array $options = [])
      *
      * @param string|array $path Path string or URL array
      * @param array $options Options array. Possible keys:
-     *   `fullBase` Return full URL with domain name
+     *   `fullBase` Return full URL with domain name. Bool or string.
```

Comment:
A slightly more detailed description and/or example would be nice so people know how exactly this option can be used to generate CDN URLs.

Reply:
Fair enough
as soon as birdy247 provides a test case I am happy to detail this in text :)